### PR TITLE
[Woo POS] Rename strings we use for POS to make them consistent

### LINKED ISSUE #11835: [Woo POS] Rename string we use for pos to make the consistent

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11835

### Description

Renames woopos strings to follow the same pattern

### Images/gif

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/cart/WooPosCartScreen.kt [hunk 1]
```diff
@@ -166,7 +166,7 @@ private fun CartToolbar(
         Spacer(modifier = Modifier.width(16.dp))
 
         Text(
-            text = stringResource(R.string.woo_pos_car_pane_title),
+            text = stringResource(R.string.woopos_car_pane_title),
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/cart/WooPosCartScreen.kt]: Maybe we should also change this to `woo_pos_cart_pane_title`, while doing the rename.

File: WooCommerce/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -11,9 +11,9 @@ Language: ar
     <string name="woopos_exit_confirmation_confirm_button">إنهاء</string>
     <string name="woopos_exit_confirmation_message">هل تريد بالتأكيد إنهاء POS؟</string>
     <string name="woopos_exit_confirmation_title">إنهاء POS</string>
-    <string name="woo_pos_car_pane_title">عربة التسوق</string>
-    <string name="woo_pos_remove_item_from_cart_content_description">إزالة عنصر من عربة التسوق</string>
-    <string name="woo_pos_checkout_button">السداد</string>
+    <string name="woopos_car_pane_title">عربة التسوق</string>
```
Comment (backwardstruck) [inline WooCommerce/src/main/res/values-ar/strings.xml]: I think we also meant to have this be `woopos_cart_pane_title`

**Review by backwardstruck — APPROVED** [review]
This should all be fine. I did notice a couple existing typos that could optionally be fixed in this PR.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/cart/WooPosCartScreen.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/cart/WooPosCartScreen.kt]: Good catch! Fixed here - 6f71092f1bb9599080ed53d2ec08cbfd28897c8c

File: WooCommerce/src/main/res/values-ar/strings.xml (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/res/values-ar/strings.xml]: The same string as the previous comment?

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/224">19.4</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/res/values-ar/strings.xml (hunk 2 above)
Comment (backwardstruck) [reply WooCommerce/src/main/res/values-ar/strings.xml]: Not sure why I commented this twice. Sorry about that!

**Pull request merged**